tests/unit_tests/worker/devices.py

```diff
@@ -41,7 +41,7 @@ def _run_callbacks(self) -> None:
 class AdditionalStatusDevice(Device, Movable):
     def set(self, value: float) -> Status:
         status = AdditionalUpdateStatus(self)
-        return status
+        return status  # type: ignore
```

Comment:
Should: What prompted this? It seems fixable

Reply:
the bluesky error is the reason - closing the conversation as discussed